### ISSUE TITLE
My DAW can't load my pugins

### ISSUE THREAD

**Issue by p1xlized** [osxmidi/LinVst#200]
Hi i changed recently my os, and i did like always(moved files from linvst folder to /usr/bin). It didn't work. When my DAW(bitwig) started indexing the plugins an error pop up with following: "Error linvst.dll not found"

- wine 6.5
- Daw Bitwig-studio 4.0.1

That's the error as shown in DAW:
/run/media/alex/2f99cec4-9224-418e-bda4-aad1b5f4c03e/Plugins/VST/Serum_x64.so

```
com.bitwig.flt.library.metadata.reader.exception.CouldNotReadMetadataException: could not read metadata: Communications error with plugin host process
exit code:139
stdout:
PluginHost: Reading VST 2.4 plugin metadata for /run/media/alex/2f99cec4-9224-418e-bda4-aad1b5f4c03e/Plugins/VST/Serum_x64.so

```
does someone has an idea.

i have tried lin-vst-X and still nothing.

UPD. wine version is 6.15 , that was my error sorry for that.

**Comment by osxmidi** [comment]
> Hi i changed recently my os, and i did like always(moved files from linvst folder to /usr/bin). It didn't work. When my DAW(bitwig) started indexing the plugins an error pop up with following: "Error linvst.dll not found"
> 
>     * wine 6.5
> 
>     * Daw Bitwig-studio 4.0.1
> 
> 
> That's the error as shown in DAW:
> /run/media/alex/2f99cec4-9224-418e-bda4-aad1b5f4c03e/Plugins/VST/Serum_x64.so
> 
> ```
> com.bitwig.flt.library.metadata.reader.exception.CouldNotReadMetadataException: could not read metadata: Communications error with plugin host process
> exit code:139
> stdout:
> PluginHost: Reading VST 2.4 plugin metadata for /run/media/alex/2f99cec4-9224-418e-bda4-aad1b5f4c03e/Plugins/VST/Serum_x64.so
> ```
> 
> does someone has an idea.
> 
> i have tried lin-vst-X and still nothing.

That happens when there is a (unnamed) linvst.so file somewhere in your vst folder paths.

**Comment by p1xlized** [comment]
The name of linvst.so match the windows .dll files. Usually i store them on my other drive and when i reinstall my os just install the libraries and host for them. This time it didn't work for some reason. i tried many things, still doesn't work
![Screenshot from 2021-08-30 10-10-10](https://user-images.githubusercontent.com/72890769/131352671-574492d4-e951-46e6-8add-05c53fe1963a.png)

**Comment by osxmidi** [comment]
> The name of linvst.so match the windows .dll files. Usually i store them on my other drive and when i reinstall my os just install the libraries and host for them. This time it didn't work for some reason. i tried many things, still doesn't work
> ![Screenshot from 2021-08-30 10-10-10](https://user-images.githubusercontent.com/72890769/131352671-574492d4-e951-46e6-8add-05c53fe1963a.png)

Bitwig can be strange.

I've had a linvst.so file (linvst binary unzipped in the Download folder) in the Downloads folder and Bitwig tried to load it causing a linvst.dll error like your one.

The Downloads folder was not in the vst path that I set in Bitwig.

The only thing I can think of is that Bitwig was scanning my Downloads folder of it's own accord.

So I moved the unzipped linvst folder (that was in the Downloads folder) to my home folder and Bitwig didn't scan the linvst.so file.

If a Daw scans linvst.so (from say a linvst binary unzipped folder) then it will produce a linvst.dll error not found because there is no linvst.dll, linvst.so hasn't been renamed to match a genuine vst dll file name.

So that might be throwing Bitwig into a spin.

I'd try moving the folder that contains linvst.so to your home folder.

Also there might be problems with the Serum Bitwig scan, so maybe install linvst again (to /usr/bin) and then redo the name conversion in your vst paths (linvstconvert).

I didn't have
